### ISSUE TITLE
Leaflet 1.9 compatibility

### ISSUE THREAD

**Issue by eusonlito** [slutske22/leaflet-spline#1]
Hello! I'm trying to install this plugin in Leaflet 1.9 but is only compatible with 1.7.1?

https://github.com/slutske22/leaflet-spline/blob/5e7d64fba481be62d8eced51e1b020c831daf398/package.json#L24

```
npm install leaflet-spline --save

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: leaflet@1.9.3
npm ERR! node_modules/leaflet
npm ERR!   dev leaflet@"^1.9.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer leaflet@"1.7.1" from leaflet-spline@0.1.3
npm ERR! node_modules/leaflet-spline
npm ERR!   leaflet-spline@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/lito/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lito/.npm/_logs/2023-01-05T08_51_29_990Z-debug-0.log
```

Thanks!

**Comment by slutske22** [comment]
Looks like I need to update the peer dependencies.  Not sure when I'll have time to do this and republish.  For now just install with the `--legacy-peer-deps` flag

**Comment by slutske22** [comment]
Should be fixed with new push of 0.2.0.  Please reinstall and check it out, report any issues, thanks!
